packages/space-opera/src/test/materials_panel/materials_panel_test.ts

```diff
@@ -75,20 +76,38 @@ describe('material panel test', () => {
     document.body.removeChild(preview);
   });
 
-  it('selector reflects materials in GLTF, including defaults where undefined',
-     async () => {
-       panel.selectedMaterialIndex = 0;
-       await panel.updateComplete;
-       expect(panel.selectedBaseColor).toEqual([1, 0, 1, 1]);
-       expect(panel.selectedRoughnessFactor).toEqual(0.2);
-       expect(panel.selectedMetallicFactor).toEqual(1);
+  describe('Given a model without any materials', () => {
+    beforeEach(async () => {
+      reduxStore.dispatch(dispatchGltfUrl(VASE_NO_TRIANGLE_GLTF_PATH));
```

Comment:
Do you mean no material instead of no triangle? Also, what's the difference between the new GLB and glTF files?

Reply:
I sure do mean material.
No difference between the 2 files. I can probably keep only a single one of the 2 formats.